roles/network_plugin/calico/tasks/main.yml

```diff
@@ -52,6 +42,16 @@
   changed_when: false
   when: "{{ use_hyperkube_cni|bool }}"
```

Comment:
I've updated the patch and replaced "cp" with "rsync". Could someone pls re-trigger CI?